.changeset/lovely-impalas-sell.md

```diff
@@ -0,0 +1,16 @@
+---
+'@backstage/plugin-search-backend-module-explore': patch
+---
+
+Allows for an optional token manager to authenticate requests from the collator to the explore backend. For example:
+
+```
```

Comment:
A suggestion could be to use `diff` here to highlight the new line passing a token manager, for example:
https://github.com/backstage/backstage/blob/6ea7d7deb8ca466763aed204fa257cac02c92cb5/docs/releases/v1.2.0-changelog.md?plain=1#LL523C6-L523C10